src/modules/tasklist.js

```diff
@@ -0,0 +1,58 @@
+import Task from './task.js';
+
+export default class TaskList {
+  constructor() {
+    this.localData = JSON.parse(localStorage.getItem('tasklist')) || [];
+  }
+
+  generateTodoIndex() {
+    return this.localData.length + 1;
+  }
+
+  refreshIndex() {
+    this.localData.forEach((task, key) => {
+      task.index = key + 1;
+    });
+    this.localStrore();
+  }
+
+  generateTaskList() {
+    let content = '';
+    this.localData.forEach((task) => {
+      content += task.renderLi();
+    });
+    return content;
+  }
+
+  addTask(description) {
+    this.localData.push(new Task(description, this.generateTodoIndex(), false));
+    this.refreshIndex();
+  }
+
+  removeItem(index) {
+    this.localData.splice(index, 1);
+    this.refreshIndex();
+  }
+
+  removeItem2(index) {
+    this.localData.splice(index, 1);
```

Comment:
- [ ] Unable to remove task from list because the delete button is not rendering correctly on the page. Please ensure to improve upon this.

**Buttons not displaying correctly**
![image](https://user-images.githubusercontent.com/12745474/193183694-d5bbb304-c523-4331-bafd-453ef619dbf5.png)